src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,47 @@
+package racingcar.domain;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Car {
+
+    private static final int MAX_NUMBER_RANGE = 9;
+    private static final int MIN_NUMBER_RANGE = 0;
+    private static final int MOVE_CONDITION = 4;
+
+    private final String name;
+    private int position = 0;
+
+    private Car(String name) {
+        this.name = name;
+    }
+
+    public static List<Car> listOf(String[] carNames) {
+        return Arrays.stream(carNames).map(Car::new).collect(Collectors.toList());
```

Comment:
```suggestion
        return Arrays.stream(carNames)
          .map(Car::new)
          .collect(Collectors.toList());
```
stream을 표현할 땐 stream 이후로 행위 하나하나에 대해 개행을 해서 가독성을 증진시키는 편입니다.